v3/lints/rfc/lint_basic_constraints_not_critical.go

```diff
@@ -60,7 +60,6 @@ func (l *basicConstCrit) Execute(c *x509.Certificate) *lint.LintResult {
 		} else {
 			return &lint.LintResult{Status: lint.Error}
 		}
-	} else {
-		return &lint.LintResult{Status: lint.NA}
 	}
+	return &lint.LintResult{Status: lint.Fatal}
```

Comment:
```suggestion
	return &lint.LintResult{Status: lint.Error}
```

In general, `Fatal` denotes that something has gone wrong in the lint itself (parsing errors, type cast errors, etc.) although I'm sure that you can find an errant exception to this somewhere in the codebase.

Reply:
Error is more appropriate. Maybe add an error message to distinguish between the two errors in the execute method? A commit for this follows soon.